extensions/panache/panache-common/deployment/src/main/java/io/quarkus/panache/common/deployment/ByteCodeType.java

```diff
@@ -53,6 +53,10 @@ public String internalName() {
         return type.getInternalName();
     }
 
+    public boolean isPrimitive() {
+        return type().getSort() < 9;
```

Comment:
Can you change 9 to org.objectweb.asm.Type#ARRAY just so it is obvious what is going on? Or maybe even <=DOUBLE to make it even clearer?

Reply:
can do.